pkg/selinux/setroubleshoot-view.jsx

```diff
@@ -458,7 +458,7 @@ export class SETroubleshootPage extends React.Component {
             <Modifications
                 title={ _("System Modifications") }
                 permitted={ this.props.selinuxStatus.permitted }
-                shell={ this.props.selinuxStatus.shell }
+                shell={ "semanage import <<EOF\n" + this.props.selinuxStatus.shell + "EOF" }
```

Comment:
Is the .shell property guaranteed to always end with a newline? Perhaps it'd be safer to use `.trim()` and explicitly add it?

Reply:
The shell property is direct output of `semanage export` and it always ends with a newline. So we should be able to count on it, but for double safety I can add it there.

Reply:
Updated